index.html

```diff
@@ -1103,6 +1103,31 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card END ________  -->
 
+        <!-- ________ *kareem* card START ________ -->
+        <div class="card">
+          <p class="name">Kareem</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/CreamCheese-dev" target="_blank"></a>
```

Comment:
Hey @CreamCheese-dev Just went through your PR commit! 

Everything looks good except that in this line linking your GitHub profile, Please provide your user name between the `<a href="https://github.com/CreamCheese-dev" target="_blank">CreamCheese-dev</a>` tag too as this would be good to redirect.